setup.py

```diff
@@ -32,5 +32,11 @@
         'Topic :: Software Development',
         'Topic :: Utilities',
     ],
-    test_suite='tests'
+    test_suite='tests',
+    package_urls={
+        'Funding': 'https://pybee.org/contributing/membership/',
+        'Documentation': 'http://batavia.readthedocs.io/en/latest/',
+        'Tracker': data['bugs']['url'],
```

Comment:
A question: where is this being sourced from?

Reply:
The `data` comes from the `package.json` in the repo. Batavia is semi-unique in that is has two package-management-config-files. As best we can, de-deduping between them is great. Sadly, `npm` does not yet support a donate link.

Reply:
The use of package.json will change as soon as I can land PR #705 - that will involve allow packaging Batavia as Python code, rather than Javascript.